[2025-02-19T15:35:38.715+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-19T15:35:38.729+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: covid_dag.import_file scheduled__2025-02-19T14:00:00+00:00 [queued]>
[2025-02-19T15:35:38.735+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: covid_dag.import_file scheduled__2025-02-19T14:00:00+00:00 [queued]>
[2025-02-19T15:35:38.736+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-19T15:35:38.747+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): import_file> on 2025-02-19 14:00:00+00:00
[2025-02-19T15:35:38.755+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=516) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-19T15:35:38.757+0000] {standard_task_runner.py:72} INFO - Started process 520 to run task
[2025-02-19T15:35:38.757+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'covid_dag', 'import_file', 'scheduled__2025-02-19T14:00:00+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/covid_dag.py', '--cfg-path', '/tmp/tmp_n31epc8']
[2025-02-19T15:35:38.759+0000] {standard_task_runner.py:105} INFO - Job 15: Subtask import_file
[2025-02-19T15:35:38.803+0000] {task_command.py:467} INFO - Running <TaskInstance: covid_dag.import_file scheduled__2025-02-19T14:00:00+00:00 [running]> on host 539a2c5d6322
[2025-02-19T15:35:38.865+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='covid_dag' AIRFLOW_CTX_TASK_ID='import_file' AIRFLOW_CTX_EXECUTION_DATE='2025-02-19T14:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-19T14:00:00+00:00'
[2025-02-19T15:35:38.866+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-19T15:35:39.574+0000] {warnings.py:112} WARNING - /opt/***/dags/covid_dag.py:8: DtypeWarning: Columns (0) have mixed types. Specify dtype option on import or set low_memory=False.
  df = pd.read_csv("https://www.data.gouv.fr/fr/datasets/r/5c4e1452-3850-4b59-b11c-3dd51d7fb8b5")

[2025-02-19T15:35:39.691+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-19T15:35:39.706+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-19T15:35:39.706+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=covid_dag, task_id=import_file, run_id=scheduled__2025-02-19T14:00:00+00:00, execution_date=20250219T140000, start_date=20250219T153538, end_date=20250219T153539
[2025-02-19T15:35:39.773+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-19T15:35:39.786+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-19T15:35:39.790+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
